icons/saxobank.svg

```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Saxo Bank</title><path d="M2.517 6.783C1.17 6.783.284 7.597.284 8.59c0 1.97 3.229 1.247 3.229 2.376 0 .424-.439.692-1.065.692-.683 0-1.124-.37-1.309-.936L0 11.085c.266.978 1.193 1.608 2.412 1.608 1.333 0 2.317-.752 2.317-1.784 0-2.041-3.252-1.376-3.252-2.404 0-.445.411-.695.966-.695.602 0 .918.289 1.082.64l1.124-.417c-.301-.772-1.124-1.25-2.132-1.25ZM21 6.784c-1.684 0-2.993 1.33-2.993 2.955s1.309 2.955 2.993 2.955c1.685 0 3-1.33 3-2.955s-1.315-2.955-3-2.955zm-12.978.09-2.439 5.73H6.83l.495-1.202h2.4l.493 1.196.784-1.157-1.945-4.568zm3.95 0 1.87 2.742-2.031 2.987h1.339l1.377-2.02 1.342 2.02h1.374l-2.042-3.008 1.843-2.722h-1.336l-1.193 1.768-1.16-1.768ZM21 7.865c1.002 0 1.763.784 1.763 1.872 0 1.089-.761 1.873-1.763 1.873-.986 0-1.756-.785-1.756-1.873s.77-1.872 1.756-1.872zm-12.477.62.79 1.92H7.735Zm6.358 6.84a.85.85 0 0 0-.345.068.58.58 0 0 0-.24.19.471.471 0 0 0-.09.284.41.41 0 0 0 .132.317.83.83 0 0 0 .359.177l.208.054a1.1 1.1 0 0 1 .173.056.31.31 0 0 1 .116.08.181.181 0 0 1 .042.118.22.22 0 0 1-.048.138.306.306 0 0 1-.128.092.511.511 0 0 1-.191.032.533.533 0 0 1-.188-.03.28.28 0 0 1-.195-.255h-.327a.57.57 0 0 0 .095.31.56.56 0 0 0 .246.193c.105.045.23.067.372.067a.936.936 0 0 0 .376-.069.499.499 0 0 0 .32-.475.46.46 0 0 0-.044-.208.48.48 0 0 0-.121-.151.727.727 0 0 0-.174-.104 1.139 1.139 0 0 0-.2-.065l-.173-.044a1.11 1.11 0 0 1-.112-.033.491.491 0 0 1-.101-.049.225.225 0 0 1-.072-.071.204.204 0 0 1 .014-.222.262.262 0 0 1 .114-.086.46.46 0 0 1 .178-.032.41.41 0 0 1 .241.066.244.244 0 0 1 .105.182h.32a.476.476 0 0 0-.087-.274.56.56 0 0 0-.231-.188.82.82 0 0 0-.344-.068zm-7.595.026v1.837h.332v-1.837zm.947 0v1.837h.332v-1.251h.015l.866 1.251h.296v-1.837h-.331v1.253h-.016l-.864-1.253Zm-5.528 0v1.838h.744c.14 0 .257-.022.35-.065a.474.474 0 0 0 .208-.18.483.483 0 0 0 .069-.256.47.47 0 0 0-.058-.238.411.411 0 0 0-.335-.213v-.018a.473.473 0 0 0 .162-.07.369.369 0 0 0 .116-.13.395.395 0 0 0 .043-.191.457.457 0 0 0-.066-.244.451.451 0 0 0-.198-.17.781.781 0 0 0-.332-.063zm1.913 0v1.838H5.82v-.279h-.87l.001-.502h.8v-.28h-.8v-.498h.862v-.279zm5.62 0 .646 1.838h.411l.648-1.838h-.366l-.477 1.446h-.02l-.478-1.446zm2.2 0v1.838h1.201v-.279h-.87l.001-.502h.8v-.28h-.8v-.498h.863v-.279zm3.599 0v.28h.568v1.558h.33V15.63h.568v-.279zm2 0v1.838h1.202v-.279h-.87v-.502h.801v-.28h-.8v-.498h.862v-.279zm1.788 0v1.838h.623a.98.98 0 0 0 .479-.11.74.74 0 0 0 .306-.317c.07-.137.106-.302.106-.493s-.035-.355-.106-.492a.736.736 0 0 0-.302-.316.952.952 0 0 0-.47-.11zm-16.787.275h.336c.099 0 .173.023.222.069a.23.23 0 0 1 .075.176c0 .054-.013.1-.04.14a.263.263 0 0 1-.108.09.37.37 0 0 1-.155.032h-.33Zm17.12.013h.283a.64.64 0 0 1 .312.07.443.443 0 0 1 .191.208.83.83 0 0 1 .065.35.835.835 0 0 1-.065.352.444.444 0 0 1-.193.211.658.658 0 0 1-.32.07h-.273Zm-17.12.734h.367c.068 0 .128.012.177.038a.274.274 0 0 1 .156.251.227.227 0 0 1-.08.18c-.054.046-.141.069-.262.069h-.358Z"/></svg>
```

Comment:
By using `Logo/Digital use/Cropped/SAXO_be_invested__saxo_blue_logo_rgb.svg` from the Zip file found at https://www.home.saxo/about-us/media-center, I got the following differences (black being you version over the original in blue):
![image](https://github.com/simple-icons/simple-icons/assets/12817388/5e858d72-c9da-4182-8a28-c6192e83498d)

Reply:
730MB :scream: